packages/stream-tile-handler/src/__tests__/tile-document.test.ts

```diff
@@ -615,6 +615,82 @@ describe('TileDocumentHandler', () => {
     await expect(makeCommit).rejects.toThrow(/Exactly one controller must be specified/)
   })
 
+  it('prohibit controllers updated to invalid values', async () => {
+    const deepCopy = (o) => StreamUtils.deserializeState(StreamUtils.serializeState(o))
+    const tileDocumentHandler = new TileDocumentHandler()
+
+    const genesisCommit = (await TileDocument.makeGenesis(context.api, {
+      test: 'data',
+    })) as SignedCommitContainer
+    await context.ipfs.dag.put(genesisCommit, FAKE_CID_1)
+    const payload = dagCBOR.decode(genesisCommit.linkedBlock)
+    await context.ipfs.dag.put(payload, genesisCommit.jws.link)
+    // apply genesis
+    const genesisCommitData = {
+      cid: FAKE_CID_1,
+      type: CommitType.GENESIS,
+      commit: payload,
+      envelope: genesisCommit.jws,
+    }
+    const genesisState = await tileDocumentHandler.applyCommit(genesisCommitData, context)
+
+    // make a first update
```

Comment:
do we need this first update?  Can we remove it to simplify and focus the test?

Reply:
You are correct - first update is not necessary. Removed.